integration/hive/hive-openlineage-hook/src/main/java/io/openlineage/hive/client/HiveOpenLineageConfig.java

```diff
@@ -25,13 +27,16 @@ public class HiveOpenLineageConfig extends OpenLineageConfig<HiveOpenLineageConf
   @Setter private JobConfig job;
 
   public HiveOpenLineageConfig(
-      TransportConfig transportConfig,
-      FacetsConfig facetsConfig,
-      DatasetConfig datasetConfig,
-      CircuitBreakerConfig circuitBreaker,
-      Map metricsConfig,
-      JobConfig job) {
-    super(transportConfig, facetsConfig, datasetConfig, circuitBreaker, metricsConfig);
+          TransportConfig transportConfig,
+          FacetsConfig facetsConfig,
+          DatasetConfig datasetConfig,
+          CircuitBreakerConfig circuitBreaker,
+          Map metricsConfig,
+          RunConfig runConfig,
+          io.openlineage.client.job.JobConfig jobConfig,
+          JobConfig job
```

Comment:
According to `import io.openlineage.client.job.JobConfig`, these fields are just the same class

Reply:
`jobConfig` is `io.openlineage.client.job.JobConfig` while `job` is `io.openlineage.hive.client.JobConfig` so the second one is for hive specific config